libbeat/outputs/elasticsearch/elasticsearch.go

```diff
@@ -91,7 +183,7 @@ func NewConnectedClient(cfg *common.Config) (*Client, error) {
 	}
 
 	for _, client := range clients {
-		err = client.Connect(client.timeout)
+		err = client.Connect()
```

Comment:
The way connection timeout works has changed a little. One sets/configures the timeouts when creating the client instance by now. The old implementation os passing the timeout to `Connect` did duplicate effort of storing/remembering these timeouts, besides always passing the same value already used to initialize the client. Plus, for applying the timeout on Connect, clients have had to adapt and reset config or actual client instances which have already been configured with the same values.
=> removed timeout parameter